grpc/proto/v1/tenants.proto

```diff
@@ -16,10 +16,9 @@ enum TenantActivityStatus {
 
 message TenantsGetRequest {
   string collection = 1;
-  bool is_consistent = 2;
   // we might need to add a tenant-cursor api at some point, make this easily extendable
   oneof params {
-    TenantNames names = 3;
+    TenantNames names = 2;
```

Comment:
**thought:** that won't be backward compatible, do we wanna [reserve](https://protobuf.dev/programming-guides/proto3/#fieldreserved)  `is_consistent` and keep

Reply:
as discussed - we only added this during the 1.25 cycle so BC does not matter here